dbt_project.yml

```diff
@@ -18,6 +18,358 @@ clean-targets: # directories to be removed by `dbt clean`
 
 vars:
   seed_source: true
+  synthea_tables: &synthea_config
+    +enabled: true
+    +schema: synthea_seeds    
+    +docs:
+      node_color: '#69AED5'
+    allergies:
+      +column_types:
+        START: date
+        STOP: date
+        PATIENT: varchar
+        ENCOUNTER: varchar
+        CODE: varchar
+        SYSTEM: varchar
+        DESCRIPTION: varchar
+        TYPE: varchar
+        CATEGORY: varchar
+        REACTION1: varchar
+        DESCRIPTION1: varchar
+        SEVERITY1: varchar
+        REACTION2: varchar
+        DESCRIPTION2: varchar
+        SEVERITY2: varchar
+    careplans:
+      +column_types:
+        Id: varchar
```

Comment:
@wylie102 for the moment i didn't add uuid casts, but i think could change the types here if we want to use uuid instead of string for the synthea IDs.